epoxy-sample/src/main/java/com/airbnb/epoxy/MainActivity.java

```diff
@@ -38,6 +38,11 @@ protected void onCreate(Bundle savedInstanceState) {
     recyclerView.setHasFixedSize(true);
     recyclerView.addItemDecoration(new VerticalGridCardSpacingDecoration());
     recyclerView.setAdapter(adapter);
+
+    // Many color models are shown on screen at once. The default recycled view pool size is
+    // only 5, so we manually set the pool size to avoid constantly creating new views when
+    // the colors are randomized
+    recyclerView.getRecycledViewPool().setMaxRecycledViews(R.layout.model_color, 50);
```

Comment:
Also through in this optimization. It will make the sample run better, and is also a nice trick to show.